concepticondata/conceptlists/Zalizniak-2020-2591.tsv

```diff
@@ -0,0 +1,2592 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_GLOSS	CONCEPTICON_ID	SHIFTS	SHIFT_URL	DEGREE	WEIGHTED_DEGREE
+Zalizniak-2020-2591-1	1	to beat, hit	BEAT	1665	1:→:20 1:→:16 1:—:15 1:→:11 1:→:10 1:→:7 1:→:7 1:→:7 1:→:6 1:→:6 1:→:6 1:—:5 1:→:5 1:↔:5 1:→:5 1:↔:4 1:→:4 1:↔:4 1:→:3 1:→:2 1:→:2 1:→:2 1:—:2 1:→:2 1:→:2 1:→:2 1:—:2 1:→:2 1:→:2 1:→:1 1:→:1 1:—:1 1:→:1 1:→:1 1:—:1 1:—:1 1:→:1 1:—:1 1:→:1 1:→:1 1:→:1 1:→:1 1:→:1	shift692 shift695 shift691 shift382 shift2393 shift1423 shift4179 shift2337 shift3945 shift337 shift2161 shift2123 shift1458 shift170 shift693 shift1226 shift345 shift2931 shift3359 shift2737 shift2091 shift171 shift2713 shift1230 shift3311 shift2090 shift2893 shift1240 shift2960 shift2336 shift3226 shift2680 shift3232 shift2340 shift4041 shift737 shift2187 shift989 shift694 shift3652 shift1228 shift2339 shift1265	43	180
```

Comment:
There is actually no clear case on how the combination of _hit_ and _beat_ is mapped across lists. In Bodt-2019-664-36 it is mapped to HIT, in Bulakh-2013-870-598 it is mapped to BEAT, in Joo-2019-100-36 it is mapped to STRIKE OR BEAT. I'll change it to STRIKE OR BEAT because it seems to be the most recent choice.